docs/source/tutorials/Start A New Project.md

```diff
@@ -0,0 +1,264 @@
+# Start A New Project
+
+If developers want to start a new project(Such as paper reproduction and finetune task), how can we use Libai to achieve the least code work.
+
+The advantages of using Libai to develop a task are as follows:
+- Avoid repeating a lot of work, many functions of Libai can be directly inherited in projects.
+- We can quickly reproduce the experiments we have run, because Libai automatically saves the configuration file.
+- Various information will be output during training, such as remaining training time, current iter, throughput, loss information and current learning rate.
+- Only a few parameters need to be set for distributed training.
+
+
+## Introduce
+Let's use Bert_Finetune task as an example to introduce Libai, The complete project is [here](https://github.com/Oneflow-Inc/libai/tree/main/projects/QQP).
+
+For start a new Libai project, our main work is as follows:
+
+1. Prepare the file named `config.py`, relevant info as follows, you can refer to the [config.py](https://github.com/Oneflow-Inc/libai/blob/main/projects/QQP/configs/config_qqp.py).
+    - This file is independent of the default config in Libai.
+    - Contains the definition of task related Class, such as `Tokenizer` and `Dataset`.
+    - Contains the relevant parameters of the task.
+    - Defining related class with `LazyCall`, this method does not call the object, but returns a dict.
+    - Many supporting objects have been imported from the `configs.common`, it can greatly reduce the workload.
```

Comment:
- The definition of task related Class, such as `Tokenizer` and `Dataset`.
- The relevant parameters of the task.
- Related class defined with `LazyCall` which returns a dict instead of calling the object.
- Built-in modules imported from the `configs.common`, which can greatly reduce the workload.

或者这样，简洁一些

Reply:
ok